tests/console/ncurses_extended.pm

```diff
@@ -0,0 +1,142 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Runs several ncurses tests from ncurses package
+# - Text attributes
+# - Colors
+# - Special characters
+# - menus
+# - panels
+# - forms
+# - background combinations
+# - key capture
+# - window movements
+
+# Maintainer: Ivan Lausuch <ilausuch@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils qw(zypper_call zypper_ar);
+use version_utils qw(is_pre_15 is_sle is_tumbleweed is_leap);
+use registration qw(add_suseconnect_product remove_suseconnect_product get_addon_fullname);
+
+sub send_key_n_times {
+    my ($key, $count) = @_;
+
+    for my $i (1 .. $count) {
+        send_key $key;
+    }
+}
```

Comment:
I suggest instead to simply write `send_key $key for (1 .. $count)` where needed. No need for the function and no need for the iterator variable.